### PR TITLE
Edit article

### PR DESCRIPTION
**Pull request by Drapegnik**
## add validation
screen shot 2018-04-08 at 20 44 00

## fix `create` & `update` for new api format
screen shot 2018-04-08 at 20 44 25

## localize edit article form

### CODE REVIEW

**Comment by Drapegnik** [comment]
[slug validation on Frontend (check out a regexp in utils/validation on the Backend)](https://trello.com/c/d73zIlFI/136-slug-validation-on-frontend-check-out-a-regexp-in-utils-validation-on-the-backend)

**Review by UladBohdan — DISMISSED** [review]
I can't really review that. Granting my approve so that you have this nice green section when merging:)

**Comment by UladBohdan** [comment]
Also updated i18n spreadsheet.
@Drapegnik if you add new tabs please don't start with empty ones, duplicate an existent - that helps to keep nice and unified formatting across the tabs.

File: routes.js [hunk 1]
```diff
@@ -33,6 +34,8 @@ const NAVBAR_ROUTES = [
       const [_, __, route, mode] = path.split('/');
       return route === 'articles' && mode === 'create';
     },
+    // dirty hack to force reload page, needed to clear store
```
Comment (tyndria) [inline routes.js:36]: I have already faced one bug related to this issue 👍 I will wait for your PR merge

**Review by UladBohdan — DISMISSED** [review]
Shall we merge it?

File: components/common/layout/Header.js [hunk 2]
```diff
@@ -64,26 +64,25 @@ class Header extends Component {
...
-            <div className="navbar-item">
-              <LocaleContext.Consumer>
-                {lang => (
+        <LocaleContext.Consumer>
+          {lang => (
+            <div className={classNames('navbar-menu', { 'is-active': burgerActive })}>
+              <div className="navbar-start">
+                {/* // TODO: check permissions for routes */}
+                {NAVBAR_ROUTES.map(({ NavLink = Link, name, pattern = name, params, isActive }) => (
+                  <NavLink key={name} route={name} params={params} lang={lang}>
```
Comment (tyndria) [inline components/common/layout/Header.js:73]: Does it make sense to receive element ``<NavLink> ... </NavLink>`` from some function? I mean, code here is too nested

**Review by tyndria — DISMISSED** [review]

**Review by UladBohdan — APPROVED** [review]
ok, merging it. will fix static delivery in a separate PR

**Comment by Drapegnik** [comment]
[create article: validation & improvements](https://trello.com/c/KRB2yJV8/173-create-article-validation-improvements)

**Pull request merged**